src/helper/configs/aruntime/nginx/nginx.go

```diff
@@ -304,15 +304,32 @@ func GenerateSslCert(ctxPath string, force bool) {
 					logger.Fatal(err)
 				}
 			} else if runtime.GOOS == "linux" {
-				cmd = exec.Command("sudo", "cp", ctxPath+"/madockCA.pem", "/usr/local/share/ca-certificates/madockCA.crt")
+
+				content, err := os.ReadFile("/etc/os-release")
+				if err != nil {
+					logger.Fatal(err)
+				}
+
+				osRelease := string(content)
+
+				if strings.Contains(osRelease, "Arch Linux") {
+					cmd = exec.Command("sudo", "cp", ctxPath+"/madockCA.pem", "/etc/ca-certificates/trust-source/anchors/madockCA.crt")
+				} else {
+					cmd = exec.Command("sudo", "cp", ctxPath+"/madockCA.pem", "/usr/local/share/ca-certificates/madockCA.crt")
```

Comment:
in this case, you can create an additional variable
path : = "/usr/local/share/ca-certificates/madockCA.crt"
and write another value to it if OS == Arch Linux